EventFilter/L1TRawToDigi/plugins/implementations_stage2/MPUnpacker_0x10010033.cc

```diff
@@ -227,7 +227,7 @@ namespace l1t {
         jet.setHwEta(CaloTools::caloEta(mpEta));
         jet.setHwPhi((raw_data >> 6) & 0x7F);
         jet.setHwPt((raw_data >> 13) & 0xFFFF);
-        jet.setHwQual((raw_data >> 29) & 0x1);
+        jet.setHwQual((raw_data >> 30) & 0x1);
```

Comment:
out of curiosity, does this change mean that the current unpacker is reading the wrong quality bit?

Reply:
yes, but this is the MP unpacker, so it isn't used anywhere (except for local firmware testing). the demux unpacker correctly unpacks the jet quality already, and is used for DQM, L1 Ntuples etc.